File: 2024/java/junior/Junior3.java
import java.util.*;

public class Junior3 {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();

        int[] scores = new int[76];
        for (int i = 0; i < n; i++) {
            int score = in.nextInt();
            scores[score]++;
        }

        int pos = 0;
        for (int i = 75; i >= 0; i--) {
            if (scores[i] > 0) {
                pos++;
            }
            if (pos == 3) {
                System.out.println(i + " " + scores[i]);
                break;
            }
        }
    }
}

File: 2024/cpp/junior/junior3.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    int n;
    cin >> n;

    int scores[76];
    fill_n(scores, 76, 0);
    for (int i = 0; i < n; i++) {
        int score;
        cin >> score;
        scores[score]++;
    }

    int pos = 0;
    for (int i = 75; i >= 0; i--) {
        if (scores[i] > 0) {
            pos++;
        }
        if (pos == 3) {
            cout << i << ' ' << scores[i] << endl;
            break;
        }
    }
}